kotlin-js-store/yarn.lock

```diff
@@ -0,0 +1,1912 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why do we need to check in this file into git? Shouldn't we exclude it through `.gitignore`?

Reply:
https://classic.yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control

Reply:
This is documentation or a JS-specific project.
Here the `yarn` file is generated by Kotlin MPP, right? Is there any recommendation on this from Kotlin?
I could only find this, but I saw our friend @JavierSegoviaCordoba was active there ;) Maybe he has experience with this.

https://github.com/renovatebot/renovate/issues/13573
https://github.com/rjaros/kvision/blob/master/build.gradle.kts#L28

Reply:
yes the Kotlin plugin generates this automatically. From what I know it's part of the build process https://kotlinlang.org/docs/js-project-setup.html#yarn

Reply:
Shouldn't we exclude files that are auto-generated from the Gradle plugin?

I don't want to block this PR because of this, since I have this file checked in into multiple repos too, and not sure what the correct approach is. This is definitely not blocking, but would be good to figure out what the correct usage is.